deno.json

```diff
@@ -6,6 +6,7 @@
   "imports": {
     "hono": "jsr:@hono/hono@4.7.4",
     "@std/toml": "jsr:@std/toml@1.0.2",
+    "prom-client": "npm:prom-client@15.1.3",
```

Comment:
I would really like to avoid npm as much as possible, would it be possible for you to point to the github repo: https://github.com/siimon/prom-client/blob/v15.1.3/index.js?

You may use https://esm.sh to help you.

Using npm in deno requires enabling many many useless environment variables that I would like to get rid of at some point if we can stop using jsdom from npm.